plugins/serializers/template/README.md

```diff
@@ -0,0 +1,34 @@
+# Template Serializer
+
+The `template` output data format outputs metrics using an user defined go template.
+[Sprig](http://masterminds.github.io/sprig/) helper functions are also available.
+
+## Configuration
+
+```toml
+[[outputs.file]]
+  ## Files to write to, "stdout" is a specially handled file.
+  files = ["stdout", "/tmp/metrics.out"]
+
+  ## Data format to output.
+  ## Each data format has its own unique set of configuration options, read
+  ## more about them here:
+  ## https://github.com/influxdata/telegraf/blob/master/docs/DATA_FORMATS_OUTPUT.md
+  data_format = "template"
+
+  ## Go template which defines output format
+  template = '{{ .Tag "host" }} {{ .Field "available" }}'
```

Comment:
Can you please also add the `batch_template` option here so we have one config where all options are shown!?